nslsii/__init__.py

```diff
@@ -548,20 +545,26 @@ def subscribe_kafka_publisher(RE, beamline_name, bootstrap_servers, producer_con
     producer_config: dict
         dictionary of Kafka Producer configuration settings
 
-    Returns
```

Comment:
Maybe add the Returns section to the docstring, because this function returns `topic` now.

Reply:
done!